pymilvus/orm/utility.py

```diff
@@ -1036,3 +1036,25 @@ def transfer_replica(source_group, target_group, collection_name, num_replicas,
         >>> rgs = utility.transfer_replica(source, target, collection_name, num_replica)
     """
     return _get_connection(using).transfer_replica(source_group, target_group, collection_name, num_replicas, timeout)
+
+def flush_all(using="default", timeout=None):
```

Comment:
Add `kwargs` to asnychronous support. Also, tell the users how to use the asynchronous ability in the document comment.

Reply:
done